src/group/formats/hal.ts

```diff
@@ -0,0 +1,26 @@
+import { User } from '../../user/types';
+import { GroupMember } from '../types';
+
+export function collection(user: User, group: GroupMember[]) {
+
+  const hal: any = {
+    _links: {
+      self: {
+        href: '/user/' + user.id + '/member',
+        title: user.nickname + ' members'
+      },
+      members: [],
```

Comment:
When a HAL resource is some kind of collection, the relationship type should always be `item`